Contents/Code/plex_api_helper.py

```diff
@@ -213,7 +214,7 @@ def update_plex_item(rating_key):
 
 
 def add_media(item, media_type, media_url_id, media_file=None, media_url=None):
-    # type: (any, str, str, Optional[str], Optional[str]) -> bool
+    # type: (Type[PlexPartialObject], str, str, Optional[str], Optional[str]) -> bool
```

Comment:
why `Type[PlexPartialObject]`? instead of `PlexPartialObject` the method accepts objects, not types?

Reply:
That was suggested by a plexapi maintainer, not sure why. My question was probably poorly worded.